app.py

```diff
@@ -347,6 +347,7 @@ def user_view():
 # user view
 @app.route('/users/<id>', methods=['GET'])
 def single_user_view(id):
+    
```

Comment:
What is the use of this commit?

Reply:
to tell what has been commited.

Reply:
oh @L04DB4L4NC3R there was no use here but I did it knowingly for a reason. Not remebering it but I had to do this. or mistakenly push.
will take more care.